app/src/main/java/com/rentminder/SaveButton.kt

```diff
@@ -1,31 +1,17 @@
 package com.rentminder
 
 import android.widget.Toast
-import androidx.compose.animation.animateContentSize
```

Comment:
removed all the imports that were not being used to reduce line bloat, and clutter when looking at imports